src/components/ContactsApp.jsx

```diff
@@ -1,41 +1,27 @@
 import React from "react";
 import ContactsList from "./ContactsList/ContactsList";
 import ContactsHeader from "./ContactsList/ContactsHeader";
-import ContactCard from "./ContactCard/ContactCard";
-import Modal, { ModalContent } from "cozy-ui/react/Modal";
-import { Button } from "cozy-ui/react/Button";
 import { PropTypes } from "prop-types";
-import { Icon, Menu, MenuItem } from "cozy-ui/react";
+import ContactsIntentButton from "./Buttons/ContactsIntentButton";
+import ContactCardModal from "./Modals/ContactCardModal";
 
-const TranslatedContactCard = ({ ...props }, { t }) => (
-  <ContactCard title={t("contact_info")} {...props} />
+const ContactsHeaderWithActions = () => (
+  <ContactsHeader
```

Comment:
This render props let `<ContactsHeader />` to be agnostic about what actions can be performed, the responsibility of this is own by the parent component, here `<ContactsApp />` through `<ContactsHeaderWithActions />` component.